web/docs/advanced/accessing-app-config.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Accessing the configuration 
+---
+
+Whenever you start a Waps app, you are starting two processes.
+ - **The client process** - A React app that implements your app's frontend.
+
+    During development, this is a dev server with hot reloading. In production,
```

Comment:
> In production, it's a Node process that serves pre-built static files with environment variables embedded during the build.

This is technically incorrect since for automatic Fly deployments we are using a Docker image with a Go server that serves static files. But for other deployments, the user is free to serve the client in which-ever way they like.

Maybe say: "the static files are serve via some sort of a server" and "this is something you can choose, but usually you'd use something like Netlify or Cloudflare Pages or a simple static files server". I'm just giving you some random ideas here to help with writing.

Reply:
Thanks, I'll figure something out.

Had no idea about the go server.